package.json

```diff
@@ -577,12 +577,6 @@
                 "category": "Jupyter",
                 "enablement": "isWorkspaceTrusted"
             },
-            {
```

Comment:
Just something quick I noticed in the debug log and wanted to clean up quick (there was a warning being thrown). This command is registered twice, keeping the instance with the disconnect icon.
https://github.com/microsoft/vscode-jupyter/blob/1fb11ad859ed3d9579caf32fbd73b49b21d029e1/package.json#L668